src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.c

```diff
@@ -31,22 +33,99 @@ const ActorInit Item_Etcetera_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Item_Etcetera/ItemEtcetera_SetupAction.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Item_Etcetera/ItemEtcetera_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Item_Etcetera/ItemEtcetera_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Item_Etcetera/func_80920044.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Item_Etcetera/func_8092009C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Item_Etcetera/func_809200F8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Item_Etcetera/ItemEtcetera_Update.s")
+static s16 D_80920230[14] = {
```

Comment:
From `index = Object_GetIndex(&globalCtx->objectCtx, D_80920230[params]);` down below we see that `D_80920230` should be filled with `ObjectIds` and so should use the `ObjectID` enum from `z64object.h`